python/ray/serve/benchmarks/cluster.yaml

```diff
@@ -1,7 +1,7 @@
 cluster_name: default
-min_workers: 22
-max_workers: 22
-initial_workers: 22
+min_workers: 5
+max_workers: 5
+initial_workers: 5
```

Comment:
reduce the number of workers, serve is having some issue which bigger cluster size which i don't have the time to look into right now. https://github.com/ray-project/ray/issues/12778